src/components/basic/input/text-area/PPTextArea.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from "clsx";
+import { useState } from "react";
+
+// ** Styles **
+import "./ppTextArea.scss";
+
+// ** Types **
+interface Props {
+  value: string;
+  setValue: React.Dispatch<React.SetStateAction<string>>;
+  rows?: number;
+  placeholder?: string;
+  label?: string;
+  disabled?: boolean;
+}
+
+const PPTextArea = ({
+  value,
+  setValue,
+  rows = 4,
+  placeholder,
+  label,
+  disabled = false,
+}: Props) => {
+  // ** Data **
+  const [isActive, setIsActive] = useState<boolean>(false);
+
+  return (
+    <div
+      className={clsx(`pp-text-area pp-input pp-input-outlined`, {
+        "pp-input-active": isActive || !!value,
+        "pp-disabled": disabled,
+      })}
+    >
+      {label && <label>{label}</label>}
+
+      <textarea
+        value={value}
+        rows={rows}
+        placeholder={placeholder}
+        disabled={disabled}
+        onChange={(e) => setValue(e.target.value)}
```

Comment:
where is e coming from?

Reply:
It the `event object` thats coming back from a `change` event firing, it stores all the data about the event, in this case we are using the value in the text area